aws/standard/standard.yaml

```diff
@@ -70,29 +70,27 @@ spec:
       generateType: init
       talosVersion: ${TALOS_VERSION}
       configPatches:
-        - op: add
-          path: /machine/kubelet/extraArgs
-          value:
-            cloud-provider: "external"
         - op: add
           path: /machine/kubelet/registerWithFQDN
           value: true
         - op: add
-          path: /cluster/extraManifests
+          path: /cluster/externalCloudProvider
           value:
-            - https://raw.githubusercontent.com/kubernetes/cloud-provider-aws/${CLOUD_PROVIDER_VERSION}/manifests/rbac.yaml
-            - https://raw.githubusercontent.com/kubernetes/cloud-provider-aws/${CLOUD_PROVIDER_VERSION}/manifests/aws-cloud-controller-manager-daemonset.yaml
+            enabled: true
+            manifests:
+              - https://raw.githubusercontent.com/kubernetes/cloud-provider-aws/${CLOUD_PROVIDER_VERSION}/manifests/rbac.yaml
+              - https://raw.githubusercontent.com/kubernetes/cloud-provider-aws/${CLOUD_PROVIDER_VERSION}/manifests/aws-cloud-controller-manager-daemonset.yaml
     controlplane:
       generateType: controlplane
       talosVersion: ${TALOS_VERSION}
       configPatches:
-        - op: add
-          path: /machine/kubelet/extraArgs
-          value:
-            cloud-provider: "external"
         - op: add
           path: /machine/kubelet/registerWithFQDN
           value: true
+        - op: add
+          path: /cluster/externalCloudProvider
+          value:
+            enabled: true
```

Comment:
this should also contain same manifests as the above for the `init` node, just fyi